src/member.rs

```diff
@@ -453,15 +447,14 @@ where
         let (u_round, u_control_hash, parent_ids) = match self.store.unit_by_hash(&u_hash) {
             Some(su) => {
                 let full_unit = su.as_signable();
+                let parents = &full_unit.control_hash().parents_mask;
+                let parents = (0..parents.len()).into_iter().map(|i| parents[i]);
```

Comment:
Why not just `let parents = full_unit.control_hash().parents_mask.iter();`?

Reply:
Given what we do later, it becomes even simpler:

`let parents: Vec<_> = full_unit.control_hash().parents().collect()`